language/proto/config.go

```diff
@@ -282,7 +282,7 @@ func checkStripImportPrefix(prefix, rel string) error {
 	if !strings.HasPrefix(prefix, "/") {
 		return fmt.Errorf("proto_strip_import_prefix should start with '/' for a prefix relative to the repository root")
 	}
-	if !pathtools.HasPrefix(rel, prefix[1:]) {
+	if rel != "" && !pathtools.HasPrefix(rel, prefix[1:]) {
```

Comment:
If this ends up being repeated anywhere else, it might make sense  for `pathtools.HasPrefix` and `TrimPrefix` to support empty paths.